src/tree/SubscriptionTreeItem.ts

```diff
@@ -61,32 +60,33 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
     public async createChildImpl(context: ICreateChildImplContext): Promise<AzureTreeItem> {
         const client: CosmosDBManagementClient = createAzureClient(this.root, CosmosDBManagementClient);
-        const wizardContext: ICosmosDBWizardContext = Object.assign(context, this.root);
+        const wizardContext: IPostgresWizardContext & ICosmosDBWizardContext = Object.assign(context, this.root);
 
         const promptSteps: AzureWizardPromptStep<ILocationWizardContext>[] = [
-            new CosmosDBAccountNameStep(),
-            new CosmosDBAccountApiStep(),
+            new AzureDBAPIStep(),
             new ResourceGroupListStep()
         ];
         LocationListStep.addStep(wizardContext, promptSteps);
 
         const wizard = new AzureWizard(wizardContext, {
             promptSteps,
-            executeSteps: [
-                new CosmosDBAccountCreateStep()
-            ],
-            title: 'Create new Cosmos DB account'
+            executeSteps: [],
+            title: 'Create new Azure Database Resource'
```

Comment:
Does this wording sound ok?
```suggestion
            title: 'Create new Azure Database Server'
```